packages/web/resolvers/utils/notifications.ts

```diff
@@ -18,7 +19,10 @@ const createEmailNotification = (
   db: PrismaClient,
   user: User,
   note: EmailNotificationCreationType,
+  digestEmailConfiguration: DigestEmailConfiguration,
 ) => {
+  if (digestEmailConfiguration === DigestEmailConfiguration.OFF) return
```

Comment:
I kinda think rather than avoiding creating the (email) notification record based on this, we should suppress sending the email based on the config preference. It's kind of a corner case, but if you turn off email notifications an hour before the email goes out, you'd expect the email not to be sent vs. this approach makes it so that you'd still get your digest email, it just won't contain the last hour of activity. The approach of collecting the notification records but not sending an email will also serve us for the weekly digest case where we still want to collect the records but only send the digest at the end of the week.

Reply:
Done!